interface/wx/webview.h

```diff
@@ -670,39 +731,79 @@ class wxWebView : public wxControl
+        @endcode
+
+        @param name Name of the message handler that can be used from javascript
+        @return @true if the handler could be added, @false if it could not be added
+
+        @see RemoveScriptMessageHandler()
+
+        @note The Edge (Chromium) backend only supports a single message handler and
+            the IE backend does not support script message handlers.
+
+        @since 3.1.5
     */
-    virtual void SetEditable(bool enable = true) = 0;
+    virtual bool AddScriptMessageHandler(const wxString& name);
 
     /**
-        Set the displayed page source to the contents of the given string.
-        @param html    The string that contains the HTML data to display.
-        @param baseUrl URL assigned to the HTML data, to be used to resolve
-                    relative paths, for instance.
-        @note When using @c wxWEBVIEW_BACKEND_IE you must wait for the current
-              page to finish loading before calling SetPage(). The baseURL
-              parameter is not used in this backend and the edge backend.
+        Remove a script message handler with the given name that was previously added via
+        AddScriptMessageHandler().
+
+        @return @true if the handler could be removed, @false if it could not be removed.
+
+        @see AddScriptMessageHandler()
+
+        @since 3.1.5
     */
-    virtual void SetPage(const wxString& html, const wxString& baseUrl) = 0;
+    virtual bool RemoveScriptMessageHandler(const wxString& name);
 
     /**
-        Set the displayed page source to the contents of the given stream.
-        @param html    The stream to read HTML data from.
-        @param baseUrl URL assigned to the HTML data, to be used to resolve
-                    relative paths, for instance.
+        Injects the specified script into the webpage’s content.
+
+        @param javascript The java script code to add
+        @param injectionTime Specifies when the script could will be
+            executed.
+        @return Returns true if the script was added successfully
+
+        @note Please not that this is unsupported by the IE backend and
+            the Edge (Chromium) backend does only support wxWEBVIEW_INJECT_AT_DOCUMENT_START.
+
+        @see RemoveAllUserScripts()
+
+        @since 3.1.5
     */
-    virtual void SetPage(wxInputStream& html, wxString baseUrl);
+    virtual bool AddUserScript(const wxString& javascript,
+        wxWebViewUserScriptInjectionTime injectionTime = wxWEBVIEW_INJECT_AT_DOCUMENT_START);
 
     /**
-        Stop the current page loading process, if any.
-        May trigger an error event of type @c wxWEBVIEW_NAV_ERR_USER_CANCELLED.
-        TODO: make @c wxWEBVIEW_NAV_ERR_USER_CANCELLED errors uniform across ports.
+        Removes all user scripts from the web view.
+
+        @see AddUserScript()
+
+        @since 3.1.5
     */
-    virtual void Stop() = 0;
+    virtual void RemoveAllUserScripts();
 
     /**
         @name Clipboard
```

Comment:
Is this comment used for anything?

Reply:
These @name comments group the methods in the class documentation member list.